events/wc_dark_cult_events.txt

```diff
@@ -1114,6 +1170,232 @@ narrative_event = {
 				character_event = { id = MNM.10030 }
 			}
 			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5041 days = 7 }
+			character_event = { id = WCDAC.5045 years = 5 }
+		}
+	}
+}
+character_event = {
+	id = WCDAC.5041
```

Comment:
Garona is OP. Able to kill from prison.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79260967-3195f600-7ea0-11ea-816d-e7ea740854cb.png)

</details>

Reply:
you can imprison people with travelling trait?

Reply:
> you can imprison people with travelling trait?

As you can see.